src/node_sea.cc

```diff
@@ -161,6 +191,54 @@ void IsExperimentalSeaWarningNeeded(const FunctionCallbackInfo<Value>& args) {
       sea_resource.flags & SeaFlags::kDisableExperimentalSeaWarning));
 }
 
+void GetCodeCache(const FunctionCallbackInfo<Value>& args) {
+  if (!IsSingleExecutable()) {
+    return;
+  }
+
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  HandleScope scope(isolate);
+
+  SeaResource sea_resource = FindSingleExecutableResource();
+
+  Local<Object> buf;
+  if (!Buffer::New(
```

Comment:
I don't think we really need all the cruft of the Buffer for this? Simply creating an DataView here should be enough.

Reply:
@joyeecheung done